### PR TITLE
i101 display no events message

### LINKED ISSUE #101: display no events message
When there is no current events display a message to let the user know

### PR DESCRIPTION
**Pull request by urwrstkn8mare**
## Change Summary
* display no events message
* create axios instance for server in frontend

### Change Form
NA

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
NA

# Related Issue

- Resolve #101

### CODE REVIEW

File: client/src/views/EventsView.vue [hunk 1]
```diff
@@ -29,8 +29,15 @@
...
       />
+      <p
+        v-if="filteredEventsList.length == 0"
+        class="pt-6 mx-3 text-center font-weight-bold text-body-1"
+      >
+        No current events :(
+      </p>
     </div>
   </div>
+  <p v-else class="pt-6 mx-3">Loading...</p>
```
Comment (ChantelleKerr) [inline client/src/views/EventsView.vue:40]: Something like this could be cool :)
https://vuetifyjs.com/en/components/progress-circular/

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
Added some suggestions for UI

File: client/src/views/EventsView.vue [hunk 2]
```diff
@@ -29,8 +29,15 @@
         :isTeamAdmin="tempIsTeamAdmin"
         @edit="openEditModal"
       />
+      <p
+        v-if="filteredEventsList.length == 0"
+        class="pt-6 mx-3 text-center font-weight-bold text-body-1"
+      >
+        No current events :(
+      </p>
```
[the remark this answers is not in the document]
Comment (urwrstkn8mare) [reply client/src/views/EventsView.vue]: i did it but have a look if its good enough. i didn't go that hard on the styling.

**Review by ChantelleKerr — APPROVED** [review]
Good job! Approved

**Pull request merged**